src/Sound/Tidal/Stream.hs

```diff
@@ -655,6 +656,19 @@ ctrlResponder waits c (stream@(Stream {sListen = Just sock}))
           = add (O.ascii_to_string k) (VS (O.ascii_to_string v))
         act (O.Message _ (O.ASCII_String k:O.Int32 v:[]))
           = add (O.ascii_to_string k) (VI (fromIntegral v))
+        -- Stream playback commands
+        act (O.Message "/mute" (O.Int32 k:[]))
+          = streamMute stream (fromIntegral k)
```

Comment:
This throws a compile warning because the second parameters to `streamMute` takes anything that can be passed to `show`, and `fromIntegral` can produce more than one type, so this is ambiguous. Actually do you need a `fromIntegral` there? I'd have thought `k` would be in the `Show` class.

Reply:
Oh, yeah, that's totally right—not sure how I didn't notice the warning